clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignor.java

```diff
@@ -663,6 +663,7 @@ private void assignOwnedPartitions() {
                 String consumer = consumerEntry.getKey();
```

Comment:
Not  necessarily related to this fix, but while you're here: would you mind helping with the readability of this class by renaming the `currentAssignment` field to `previousAssignment`? Took me a few minutes to understand what was going on until I realized this map just had a misleading name

Reply:
I went to do so, but on starting I realised it's a bit of a rabbit hole that I would prefer not to go down. There are multiple other uses of the words "current" and "previous" throughout the class/es and changing this one variable would cause other lines to become more confusing eg

```
            log.info("Final assignment of partitions to consumers: \n{}", currentAssignment);
```

Reply:
fair enough. thanks for taking a look